mypy.ini

```diff
@@ -1,5 +1,35 @@
 [mypy]
-ignore_missing_imports = True
-# required to support namespace packages
-# https://github.com/python/mypy/issues/14057
+# Is the project well-typed?
+strict = False
+
+# Early opt-in even when strict = False
+warn_unused_ignores = True
+warn_redundant_casts = True
+enable_error_code = ignore-without-code
+
+# Support namespace packages per https://github.com/python/mypy/issues/14057
 explicit_package_bases = True
+
+# Disable overload-overlap due to many false-positives
+disable_error_code = overload-overlap
+
+exclude = (?x)(
+    # upstream
+	^build/
+	| ^.tox/
+ 
+ 	# local
+	)
+
+# - distutils._modified has different errors on Python 3.8 [import-untyped], on Python 3.9+ [import-not-found]
+# - All jaraco modules are still untyped
```

Comment:
Some (many?) jaraco modules are now typed, so this no longer applies.

Reply:
Many aren't typed strictly as #98 is still open, but yeah, it no longer applies

Reply:
This will improve with time anyway (both in terms of inaccurate/incomplete typing, and being marked as `py.typed`), I don't mind not including it in the base recommendation.
But my *annecdotal* experience with setuptools' specific dependencies, most weren't.

Reply:
Yeah
I guess we could adopt a tool like https://github.com/jellezijlstra/autotyping to speed up the process, wdyt @Avasam?

Reply:
Oh, sorry.
To **speedrun**\* the process. ;)

Reply:
> Yeah I guess we could adopt a tool like [jellezijlstra/autotyping](https://github.com/jellezijlstra/autotyping) to speed up the process, wdyt @Avasam?

Never tried it, but such a tool from Jelle is worth trying.

Worth mentioning that https://github.com/google/pytype (which typeshed tests agains) claims to be able to generate stub files from inference (it works from the bottom-up, unlike most type-checkers), which can then be merged back into source code. 

In any case, I'd prefer for a package to be confident in its typing before publicising itself as `py.typed`.